com.unity.netcode.gameobjects/Runtime/Transports/UTP/UnityTransport.cs

```diff
@@ -330,7 +330,21 @@ private static NetworkEndpoint ParseNetworkEndpoint(string ip, ushort port)
             /// <summary>
             /// Endpoint (IP address and port) server will listen/bind on.
             /// </summary>
-            public NetworkEndpoint ListenEndPoint => ParseNetworkEndpoint((ServerListenAddress?.Length == 0) ? Address : ServerListenAddress, Port);
+            public NetworkEndpoint ListenEndPoint
+            {
+                get
+                {
+                    if (ServerListenAddress?.Length == 0)
```

Comment:
Won't this log `Debug.LogError($"Invalid network endpoint: {ip}:{port}.");` in the case where the server listen address is empty?

I don't see an actual test case for this conditional - do we have one for an empty address versus a malformed one?

Reply:
Rather, line 339 (accessing ServerEndPoint) would log this in this case.

Reply:
Yes, if both the 'Address' and 'Listen Address' fields are left empty, it will log an error. Although that scenario was never supported before. The previous behavior if leaving 'Listen Address' empty was to use 'Address'. I can add a special case where if both are left empty, we revert to 0.0.0.0 as the listen address.

There are no specific tests for leaving the listen address empty, but that is what a lot of tests actually end up doing, since it's the default when calling `SetConnectionData` without explicitly setting a listen address.

Reply:
Pushed a change that avoids an error if both 'Address' and 'Listen Address' are empty (we'll then default to 0.0.0.0). Also added a test case for this particular scenario.